samigo/samigo-services/src/java/org/sakaiproject/tool/assessment/services/assessment/AssessmentEntityProducer.java

```diff
@@ -104,6 +113,8 @@ public class AssessmentEntityProducer implements EntityTransferrer, EntityProduc
     @Getter @Setter protected UserDirectoryService userDirectoryService;
     @Getter @Setter protected PublishedAssessmentFacadeQueriesAPI publishedAssessmentFacadeQueries;
 
+	private final LinkMigrationHelper linkMigrationHelper = ComponentManager.get(LinkMigrationHelper.class);
```

Comment:
Please avoid using the ComponentManager when possible, this class is already managed by Spring.